### ISSUE TITLE
CANS Assessment Application Critical Accessibility Issues

### ISSUE THREAD

**Issue by abelayt** [ca-cwds/cans#157]
Conducted an automated assessment of the current CANS interface in the Integration environment.  Please address these issues to pass accessibility compliance for code complete.
### PRIORITY 1 CRITICAL
**1. Aria attributes do not have valid values**
Failing Elements
<img width="394" alt="screen shot 2018-09-12 at 12 18 47 pm" src="https://user-images.githubusercontent.com/30639852/45447961-05900e80-b686-11e8-9fd0-91f0c21413d6.png">
**How to fix:** https://dequeuniversity.com/rules/axe/2.2/aria-valid-attr-value?application=lighthouse

**2. Form elements do not have associated labels**
Failing Elements
<img width="406" alt="screen shot 2018-09-12 at 12 24 23 pm" src="https://user-images.githubusercontent.com/30639852/45448284-cf06c380-b686-11e8-9771-d824c04167ac.png">
**How to Fix:** https://dequeuniversity.com/rules/axe/2.2/label?application=lighthouse

**3. Page specifies valid language**
Failing Element
<img width="398" alt="screen shot 2018-09-12 at 12 26 22 pm" src="https://user-images.githubusercontent.com/30639852/45448391-142af580-b687-11e8-8466-10329238f8ae.png">
**How to Fix:** https://dequeuniversity.com/rules/axe/2.2/html-has-lang

### PRIORITY 2 CRITICAL
Color contrast is another critical issue, however it can be address post code complete unless there is time for a developer to address it.  If so, ...
Color contrast is not satisfactory
Failing Elements
<img width="409" alt="screen shot 2018-09-12 at 12 30 14 pm" src="https://user-images.githubusercontent.com/30639852/45448616-a4693a80-b687-11e8-8a7a-644cbd0e0654.png">
**How to Fix:**  Work with DesignOps to solidify color palette with efficient contrast
https://dequeuniversity.com/rules/axe/2.2/color-contrast?application=lighthouse

______
Manual accessibility assessment have come up with additional issues, but those can be addressed after code complete milestone.

**Comment by abelayt** [comment]
These issues were resolved as part of CANS-309 therefore closing this ticket.
